types/container/config.go

```diff
@@ -3,8 +3,29 @@ package container
 import (
 	"github.com/docker/engine-api/types/strslice"
 	"github.com/docker/go-connections/nat"
+	"time"
 )
 
+// HealthConfig holds configuration settings for the HEALTHCHECK feature.
+type HealthConfig struct {
+	// Test to perform to check container is healthy.
+	// An empty slice means to inherit the default.
+	// The options are:
+	// {} : inherit healthcheck
+	// {"NONE"} : disable healthcheck
```

Comment:
No, but you can provide a command that calls the command from the base image and also performs other tests too. Having multiple healthcheck commands would make the API a lot more complicated, so we decided not to do that.